code/Test_Definitions/number-verification-device-phone-number-share.feature

```diff
@@ -66,7 +66,7 @@ Feature: Camara Number Verification API device phone number share
     And the response body complies with the OAS schema at "/components/schemas/ErrorInfo"
     Then the response status code is 401
     And the response property "$.status" is 401
-    And the response property "$.code" is "AUTHENTICATION_REQUIRED"
+    And the response property "$.code" is "UNAUTHENTICATED"
     And the response property "$.message" is "New authentication is required."
```

Comment:
A very minor fix, but to be align with the common artifact: https://github.com/camaraproject/Commonalities/blob/main/artifacts/CAMARA_common.yaml#L174C26-L174C100

```suggestion
    And the response property "$.message" is "Request not authenticated due to missing, invalid, or expired credentials."
```